exchange/events.go

```diff
@@ -41,6 +45,9 @@ func (ev *eventTracking) modifyBidsForEvents(seatBids map[openrtb_ext.BidderName
 	for bidderName, seatBid := range seatBids {
 		modifyingVastXMLAllowed := ev.isModifyingVASTXMLAllowed(bidderName.String())
 		for _, pbsBid := range seatBid.bids {
+			if ev.generateBidId {
+				pbsBid.generatedBidId = generateBidId()
```

Comment:
How about 
```suggestion
				pbsBid.generatedBidId = uuid.NewString()
```
using `github.com/google/uuid`

Reply:
@laurb9 @SyntaxNode  
Should I reuse existing `uuid.NewV4()` from "github.com/gofrs/uuid" like we do in cache (action.go)?

Reply:
I don't care which library is used. Both are actively maintained and the function signatures are identical.

Reply:
Either way, please consider having a bid id generator interface so it can be swapped out for predictable values for test.

Reply:
@SyntaxNode interface created. Looks cleaner and now it's decoupled from events. Good point!
I. reused existing "github.com/gofrs/uuid" to generate UUID, is it ok?

Reply:
That's fine. I don't care which library we use. We've seen the gofrs work just fine at scale in Prebid Cache. Thanks for the update. This does look much nicer.